src/groups/groups.repository.ts

```diff
@@ -133,4 +133,20 @@ export class GroupsRepository {
       where: { id: groupId },
     });
   }
+
+  async findGroupBannerPath(groupId: number): Promise<string | null> {
+    const g = await this.prisma.group.findUnique({
```

Comment:
바로 return 할 수 있는 변수라 큰 의미는 없는데 과한 축약은 가독성을 헤치는 요인이 됩니다

```suggestion
    const group = await this.prisma.group.findUnique({
```